### ISSUE TITLE
TypeError: Class constructor BugSplat cannot be invoked without 'new'

### ISSUE THREAD

**Issue by def0e0** [BugSplat-Git/bugsplat-node#17]
Trying to add BugSplat to the electron project (webpack + typescript template). 
At startup error appears:
```
App threw an error during load
TypeError: Class constructor BugSplat cannot be invoked without 'new'
```

To repeat this error you need to create a project as described on site [Electron Forge](https://www.electronforge.io/templates/typescript-+-webpack-template) and then add BugSplat as described [here](https://docs.bugsplat.com/introduction/getting-started/integrations/cross-platform/node.js). 
I attach an archive with the project on which the error is reproduced.
[electron_test.zip](https://github.com/BugSplat-Git/bugsplat-node/files/14178616/electron_test.zip)

**Comment by bobbyg603** [comment]
Thanks for reporting this @def0e0! I think this should be fixed by #18. We'll try and get that released asap.

**Comment by bobbyg603** [comment]
Hi @def0e0, I just tested 3.0.2 in the sample you sent and it worked for me! Please re-open this issue if you're still having trouble!
